doc/manual/src/language/syntax.md

```diff
@@ -503,6 +503,24 @@ three kinds of patterns:
     > [ 23 {} ]
     > ```
 
+    All bindings introduced by the function are in scope in the entire function expression; not just in the body. For example, the binding of an `@` pattern is available in the `?`-defaults:
+    It can therefore be used in default values.
+
+    > **Example**
+    >
```

Comment:
```suggestion
    All bindings introduced by the function are in scope in the entire function expression; not just in the body.
    It can therefore be used in default values.

    > **Example**
    >
    >  The binding of an `@` pattern is available in the `?`-defaults:
    >
```